breakout_1/breakout_1_answers.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Breakout 1: Working With Data Frames"
+subtitle: "PHUSE US Connect 2021: Introduction to R for the Statistical Programmer"
+author: "Atorus Research"
+output:
+  rmarkdown::html_document:
+    theme: lumen
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(haven)
+library(knitr)
+```
+
+## Intro
+
+We're going to work with some data from the DM and AE datasets.
```

Comment:
I think all of the intros say the same thing. I think we can skip it and assume the user knows what the data looks like

Reply:
This is breakout 1 so this is the starting point - so here we should include. In subsequent ones (like breakout 4) I have it with a "Let's pick up where we left off" and then apply the data processing that the last training finished with.